lib/ansible/modules/system/filesystem.py

```diff
@@ -23,9 +23,11 @@
 version_added: "1.2"
 options:
   fstype:
+    choices: [ "ext4", "ext4dev", "ext3", "ext2", "xfs", "btrfs", "reiserfs", "lvm"]
```

Comment:
Isn't the check made [L213](https://github.com/mscherer/ansible/blob/c19a95a315680a3f68b157054aa54c0ecc31ee28/lib/ansible/modules/system/filesystem.py#L214) sufficient ?